client/order/index.js

```diff
@@ -149,37 +150,48 @@ const DisputeNotice = ( { chargeId } ) => {
 		return;
 	}
 
+	const titleStrings = {
+		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
+		dispute_default: __(
+			// eslint-disable-next-line max-len
+			'This order has been disputed in the amount of %1$s. The customer provided the following reason: %2$s. Please respond to this dispute before %3$s.',
+			'woocommerce-payments'
+		),
+		inquiry_default: __(
```

Comment:
Missing translators comment. 
```suggestion
    // Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
	  inquiry_default: __(
```